src/Tools/dotnet-monitor/EndpointInfo/EndpointInfoExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    internal static class EndpointInfoExtensions
+    {
+        public static string GetDebuggerDisplay(this IEndpointInfo endpointInfo)
```

Comment:
This is mostly used outside of getting a Debugger Display attribute. Maybe a better name would be:
```cs
public static string GetFriendlyDisplayString(this IEndpointInfo endpointInfo)
```

Reply:
I've split the extension method to be inlined with their current usages.